src/Serilog.Sinks.Sentry/SentrySink.cs

```diff
@@ -24,12 +25,17 @@ public class SentrySink : ILogEventSink
         /// <param name="dsn">The DSN.</param>
         /// <param name="release">The release.</param>
         /// <param name="environment">The environment.</param>
-        public SentrySink(IFormatProvider formatProvider, string dsn, string release, string environment)
+        /// <param name="tags">Comma separated list of properties to treat as tags in sentry.</param>
+        public SentrySink(IFormatProvider formatProvider, string dsn, string release, string environment, string tags)
         {
             _formatProvider = formatProvider;
             _dsn = dsn;
             _release = release;
             _environment = environment;
+            if (!string.IsNullOrWhiteSpace(tags))
+            {
+                _tags = tags.Split(',');
```

Comment:
Could you please change the call to this overload with `StringSplitOptions.RemoveEmptyEntries`?
https://docs.microsoft.com/en-us/dotnet/api/system.string.split?view=netframework-4.5.1#System_String_Split_System_Char___System_StringSplitOptions_ #